README.md

```diff
@@ -38,7 +38,9 @@ to use it is via [Esy](https://esy.sh).
 }
 ```
 
-3. Reach out on the [ReasonML Discord](https://discord.gg/reasonml) if you
+3. Run `npm install` (or `yarn`) followed by just `esy` and after they finish installing and building, you should be able to use `bsb` commands via esy `esy bsb -make-world`
```

Comment:
do you think we should recommend adding `esy` as the `postinstall` script in `package.json`?

I'm a little torn on it, so just wanted to get the discussion started.

Reply:
I do yes, I actually threw my thoughts on a quick note while working on this:
```
We can have a template that adds a good .gitignore, esy.json, package.json with
a “postinstall”: “esy” script as well as pre-populated “build”: “esy bsb -make-
world” for a much more seamless npm experience
`npm install` followed by `npm build` becomes all that's needed as customary 
and then people can add their own `npm start` based on their project
```
What I am personally conflicted about is whether to have "esy" as a dev dependency in the package.json or if that's even a recommended approach